notifications.bs

```diff
@@ -98,8 +98,16 @@ associated <dfn>icon URL</dfn>, <dfn>icon resource</dfn>, <dfn>sound URL</dfn>,
 <dfn>sound resource</dfn>, <dfn>vibration pattern</dfn>, and
 <dfn>service worker registration</dfn>.
 
-<p class=note>Developers are encouraged to not convey information through an icon, sound,
-or vibration pattern that is not otherwise accessible to the end user.
+<p>A <a lt="concept notification">notification</a> <em>can</em> have an
+associated <dfn>badge URL</dfn> and <dfn>badge resource</dfn>. A <a>badge
+resource</a> <em>may</em> may be used to represent the
+<a lt="concept notification">notification</a> when there is not enough space to
+display the <a lt="concept notification">notification</a> itself. It may also be
+displayed inside the <a lt="concept notification">notification</a>.
```

Comment:
I think we need a little bit more context here with respect to how this feature relates to icon.